content/content.types.ts

```diff
@@ -19,6 +19,7 @@ export interface AirtableEmbed {
     title: string;
     id: string;
     src: string;
+    height: string;
```

Comment:
This type is also being used in the get involved content, and so adding the height has given `openRole` this error: `Property height is missing in type but required in type`. Let's take the height on `pages/get-involved` line 183, and add it to `content/get-involved`. I usually run a quick search of the codebase to see where a type is being used before making changes to it.